src/ray/gcs/gcs_server/gcs_resource_manager.h

```diff
@@ -28,8 +28,10 @@
 #include "src/ray/protobuf/gcs.pb.h"
 
 namespace ray {
+namespace syncer {
```

Comment:
Do we need this?

Reply:
Unfortunately, there is a circular dependence for now. It should be removed in the follow-up PR once we remove the pg resource reporting code path